Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -103,18 +114,35 @@ final class MessageBannerViewController: UIViewController, NibLoading {
       self.view.superview?.layoutIfNeeded()
     }
 
-    UIView.animate(withDuration: duration, delay: 0.0,
-                   options: UIView.AnimationOptions.curveEaseInOut,
-                   animations: { [weak self] in
-                    guard let self = self else { return }
-
-                    self.bottomConstraint?.constant = isHidden ? hiddenConstant : 0
-
-                    self.view.superview?.layoutIfNeeded()
-    }, completion: { [weak self] _ in
-      self?.view.isHidden = isHidden
-
-      self?.viewModel.inputs.bannerViewAnimationFinished(isHidden: isHidden)
+    UIView.animate(
```

Comment:
Mostly changing the style of completion blocks for better readability but two important things happen here:
1. When the banner is showed we focus accessibility reader on it
2. When the banner is hidden we focus accessibility on the perilously focused element (which will most likely be `Save` button)